src/test/bin/winkernel/control.cpp

```diff
@@ -690,6 +690,19 @@ QuicTestCtlEvtIoDeviceControl(
                 nullptr,
                 nullptr,
                 STRSAFE_NULL_ON_FAILURE);
+
+#if defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
+        if (UseDuoNic) {
+            QUIC_EXECUTION_CONFIG Config = Params->TestConfigurationParams.Config;
+            Status =
+                MsQuic->SetParam(
+                    nullptr,
+                    QUIC_PARAM_GLOBAL_EXECUTION_CONFIG,
+                    sizeof(Config),
+                    &Config);
+        }
```

Comment:
Change this to be closer to what user mode does?
```suggestion
        QUIC_EXECUTION_CONFIG Config = Params->TestConfigurationParams.Config;
        if (Config->Flags != QUIC_EXECUTION_CONFIG_FLAG_NONE) {
            Status =
                MsQuic->SetParam(
                    nullptr,
                    QUIC_PARAM_GLOBAL_EXECUTION_CONFIG,
                    sizeof(Config),
                    &Config);
        }
```

Reply:
LGTM!